webinterface/pages/DDA_Quant_ion.py

```diff
@@ -99,617 +90,535 @@ def generate_input_field(self, input_format: str, content: dict):
-                    The full description of the pre-processing steps and metrics
-                    calculation is available [here](https://proteobench.readthedocs.io/en/latest/modules/3-DDA-Quantification-ion-level/).
-                    """
-        )
+        st.markdown(open("pages/markdown_files/DDA_Quant_ion/introduction.md", "r").read())
         st.header("Downloading associated files")
-        st.markdown(
-            """
-                    The raw files used for this module were acquired on an Orbitrap
-                    Q-Exactive H-FX (ThermoScientific). They can be downloaded from the
-                    proteomeXchange repository PXD028735. You can download them here:
-
-                    [LFQ_Orbitrap_DDA_Condition_A_Sample_Alpha_01.raw](https://ftp.pride.ebi.ac.uk/pride/data/archive/2022/02/PXD028735/LFQ_Orbitrap_DDA_Condition_A_Sample_Alpha_01.raw),
-                    [LFQ_Orbitrap_DDA_Condition_A_Sample_Alpha_02.raw](https://ftp.pride.ebi.ac.uk/pride/data/archive/2022/02/PXD028735/LFQ_Orbitrap_DDA_Condition_A_Sample_Alpha_02.raw),
-                    [LFQ_Orbitrap_DDA_Condition_A_Sample_Alpha_03.raw](https://ftp.pride.ebi.ac.uk/pride/data/archive/2022/02/PXD028735/LFQ_Orbitrap_DDA_Condition_A_Sample_Alpha_03.raw),
-                    [LFQ_Orbitrap_DDA_Condition_B_Sample_Alpha_01.raw](https://ftp.pride.ebi.ac.uk/pride/data/archive/2022/02/PXD028735/LFQ_Orbitrap_DDA_Condition_B_Sample_Alpha_01.raw),
-                    [LFQ_Orbitrap_DDA_Condition_B_Sample_Alpha_02.raw](https://ftp.pride.ebi.ac.uk/pride/data/archive/2022/02/PXD028735/LFQ_Orbitrap_DDA_Condition_B_Sample_Alpha_02.raw),
-                    [LFQ_Orbitrap_DDA_Condition_B_Sample_Alpha_03.raw](https://ftp.pride.ebi.ac.uk/pride/data/archive/2022/02/PXD028735/LFQ_Orbitrap_DDA_Condition_B_Sample_Alpha_03.raw)
-
-                    **It is imperative not to rename the files once downloaded!**
-                    """
-        )
-        st.markdown(
-            """
-                    Download the zipped FASTA file here: <a href="/app/static/ProteoBenchFASTA_DDAQuantification.zip" download>ProteoBenchFASTA_DDAQuantification.zip</a>.
-                    The fasta file provided for this module contains the three species
-                    present in the samples and contaminant proteins
-                    ([Frankenfield et al., JPR](https://pubs.acs.org/doi/10.1021/acs.jproteome.2c00145))
-                    """,
-            unsafe_allow_html=True,
-        )
-
+        st.markdown(open("pages/markdown_files/DDA_Quant_ion/file_description.md", "r").read(), unsafe_allow_html=True)
         st.header("Input and configuration")
-
-        st.markdown(
-            """
-                   Scroll down if you want to see the public benchmark runs publicly available
-                   today.
-                    """
-        )
+        st.markdown(self.texts.ShortMessages.initial_results)
```

Comment:
I don't know why, here there is a `"` printed before the sentence in the webinterface. 
<img width="500" alt="image" src="https://github.com/Proteobench/ProteoBench/assets/8386272/0f66d174-13a7-48f9-88c7-4afb4d262e2a">